tensorflow/python/training/device_setter_test.py

```diff
@@ -77,6 +77,23 @@ def testPS2TasksWithDevice(self):
       self.assertDeviceEqual("/job:moon/task:1", w.device)
       self.assertDeviceEqual("/job:moon/task:1", w.initializer.device)
       self.assertDeviceEqual("/job:sun", a.device)
+  
+  def testPS2TasksWithCPUConstraint(self):
+     cluster_spec = tf.train.ClusterSpec({
+        "sun": ["sun0:2222", "sun1:2222", "sun2:2222"],
+        "moon": ["moon0:2222", "moon1:2222"]})
```

Comment:
It looks like the indentation is slightly inconsistent here: to satisfy our linter (which follows [these style rules](https://google.github.io/styleguide/pyguide.html)), the code should be indented two spaces at each block, and statement continuations (such as L83 and L84) should be indented four spaces.

Can you please fix that up? Then we should be good to go!